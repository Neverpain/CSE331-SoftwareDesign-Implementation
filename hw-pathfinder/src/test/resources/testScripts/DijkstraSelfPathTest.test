# Tests the output of a path from a node to itself
# using Dijkstra's algorithm is correct, prints it out

CreateGraph g

AddNode g A

FindPath g A A