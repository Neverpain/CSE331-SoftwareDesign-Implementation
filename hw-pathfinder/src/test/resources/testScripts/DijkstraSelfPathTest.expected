# Tests the output of a path from a node to itself
# using Dijkstra's algorithm is correct, prints it out

created graph g

added node A to g

path from A to A:
total cost: 0.000